src/main/java/org/craftercms/studio/impl/v1/service/site/SiteServiceImpl.java

```diff
@@ -353,110 +345,108 @@ private void insertCreateSiteAuditLog(String siteId, String siteName, String blu
 
     private void processCreatedFiles(String siteId, Map<String, String> createdFiles, String creator,
```

Comment:
Let's extract to another method so we don't have a huge lambda. Either that or add the `logExecutionTime` call to the caller